server/util.go

```diff
@@ -181,23 +181,20 @@ func CreateSDKLogger(ctx *Context, out io.Writer) (log.Logger, error) {
 	}
 
 	logLvl, err := zerolog.ParseLevel(logLvlStr)
-	if err != nil {
-		// If the log level is not a valid zerolog level, then we try to parse it as a key filter.
+	switch {
+	case err != nil:
 		filterFunc, err := log.ParseLogLevel(logLvlStr)
 		if err != nil {
 			return nil, err
 		}
-
 		opts = append(opts, log.FilterOption(filterFunc))
 
-	} else if ctx.Viper.GetBool(cmtcli.TraceFlag) {
-		// Check if the CometBFT flag for trace logging is set if it is then setup a tracing logger in this app as well.
```

Comment:
Please leave the comment, which explains the behavior.

Reply:
yeah I def  not removing comments -- eg a tool is.  I will try and figure out which and why and get it gone.

Reply:
ahhh it is in a switch.  I know how this happened.